modules/5_install/templates/helpernode_vars.yaml

```diff
@@ -46,5 +46,16 @@ setup_registry:
   release_tag: "${local_registry.ocp_release_tag}"
 %{ endif }
 
+%{ if chrony_config ~}
+chronyconfig:
+  enabled: true
+  content:
+%{ for item in chrony_config_servers ~}
```

Comment:
I've done this on install_vars.yaml because I do not want to have `chronyconfig.content` empty... 

For helpernode template, it is a bit different as I add bastion_IP anyway just after. So `chronyconfig.content` cannot be empty.